doc/configuration.rst

```diff
@@ -1025,6 +1025,27 @@ resource available on a remote computer
 Used by:
   - `FlashScriptDriver`_
 
+Dediprog
+~~~~~~~~
+A Dediprog resource is used to configure the parameters to a locally installed
+dpmcd instance. It is assumed that dpcmd is installed on the host and the
+executable is configured in:
+
+.. code-block:: yaml
+
+  tools:
+    dpcmd: '/usr/sbin/dpcmd'
+
+Arguments:
+  - vcc (int): 0 (3.5V), 1 (2.5V), 2 (1.8V)
```

Comment:
Hmm, I'd prefer to have the actual voltage in the configuration file, as this makes it notice a wrong configuration. You can map the voltages to the command line arguments in the driver.

Reply:
Fair.